docs/how_to_contribute.md

```diff
@@ -15,85 +15,117 @@ And there are a lot of domains.
+We will continue here with `la_times.py`.
+In the newly created file, add an empty parser class inheriting from `ParserProxy` and a parser version `V1` subclassing `BaseParser`.
 
 ``` python
-from fundus.parser import BaseParser
+from fundus.parser import ParserProxy, BaseParser
 
-class LosAngelesTimesParser(BaseParser):
-    pass
+
+class LATimesParser(ParserProxy):
+    class V1(BaseParser):
+        pass
 ```
 
+Internally, the `ParserProxy` maps crawl dates to specific versions (`V1`, `V2`, etc.) subclassing `BaseParser`.
+Since Fundus' parsers are written by hand and in most cases bound to the layout they were written for, Fundus takes the extra proxying step to address changes to the layout.
+Don't worry about it now.
+To add a new parser you don't even have to know why Fundus does things this way.
+Just stick to the architecture shown above, and you're good to go.
+
 ### 2. Creating a Publisher Specification
 
 Add a new publisher specification for the publisher you want to cover.
-The publisher specification links the publisher's domain, sitemap and the corresponding parser to the publisher.
+The publisher specification links information about the publisher, sources where to get the HTML to parse from, and the corresponding parser to an object used by Fundus' `Crawler`.
 
 You can add a new entry to the country-specific `PublisherEnum` in the `__init__.py` of the country section you want to contribute to, i.e. `fundus/publishers/<country_code>/__init__.py`.
-For now, we specify the publisher's domain and parser.
-We cover the publisher's sitemap in the next step.
+For now, we only specify the publisher's name, domain, and parser.
+We will cover sources in the next step.
 
 For the Long Angeles Times, we add the following entry to `fundus/publishers/us/__init__.py`.
 
 ``` python
 class US(PublisherEnum):
-    LosAngelesTimes = PublisherSpec(
+    LATimes = PublisherSpec(
+        name="Los Angeles Times",
         domain="https://www.latimes.com/",
-        parser=LosAngelesTimesParser,
+        parser=LATimesParser,
     )
 ```
 
 If the country section for your publisher did not exist before step 1, please add the `PublisherEnum` to `src/library/collection/__init__.py'`.
 
-### 3. Adding Sitemaps
+### 3. Adding Sources
+
+For your newly added publisher to work we first need to specify where to find articles - in the form of HTML - to parse.
+Fundus currently supports two methods to do so. Either by specifying RSS Feeds or sitemaps as `URLSource`.
+You do so by adding the corresponding `URLSource` objects with the `sources` parameter to the `PublisherSpec`.
 
-TODO: rewrite entire section, rename to -> add sources
-TODO: Introduce Index Maps (Delete from Google News Maps) -> In separate Sections
+There are three of them you can import from the `fundus.scraping.html` module.
```

Comment:
This line could be improved: ""Fundus provides the following types of UrSources in its module `fundus.scraping.html`  ".
Maybe add a link to Wikipedia or their technical definition in this list as well? This eases research on these.

Reply:
Done, except the wikipedia part. That seems a bit too much for me.

Reply:
Allright.